### DISCUSSION TITLE
Help Needed: Local Development Setup with IntelliJ IDEA Community Edition

### DISCUSSION THREAD

**Discussion by aryel780** [tolgee/tolgee-platform#2633] [Q&A] [answered]
I'm setting up the Tolgee project locally by following the guide provided [here](https://github.com/tolgee/tolgee-platform/wiki/Development) and using IntelliJ IDEA Community Edition (CE). However, I'm running into some issues and would appreciate any assistance to resolve them.

Here is the issue I'm having

https://github.com/user-attachments/assets/10b6f987-090b-479e-80cc-429be37310e9

**Comment by JanCizmar** [comment]
Hey! Can you please run ti with --stacktrace and send the output? Thanks!

**Reply by aryel780** [reply]
Here the output

https://github.com/user-attachments/assets/2fee4359-fd7d-4d88-b22a-9db5a65f8f26

**Reply by aryel780** [reply]
After running the frontend, it shows this error on localhost.

<img width="1394" alt="Screenshot 2024-10-30 at 2 57 03 PM" src="https://github.com/user-attachments/assets/6ffbe0f6-74ee-4a1b-bfca-31c2277640bf">

**Reply by JanCizmar** [reply]
Hey! I have no clue, how you have it configured in the run configuration dialog. Can you please share what's in the configuration dialog?

**Reply by aryel780** [reply]
<img width="1470" alt="Screenshot 2024-10-30 at 7 39 19 PM" src="https://github.com/user-attachments/assets/9564ecb6-39f6-49eb-8611-bc11d5c06c12">

<img width="1470" alt="Screenshot 2024-10-30 at 7 39 56 PM" src="https://github.com/user-attachments/assets/af58706f-aa57-45fd-bf08-ac17dad8e72a">

**Reply by JanCizmar** [reply]
You have to add `--stacktrace` to the command. Not replace the command with it.

**Reply by aryel780** [reply]
https://github.com/user-attachments/assets/1dd2dd29-4cf3-41be-8a14-8936c6ba915e

**Reply by JanCizmar** [reply, marked as answer]
Hey! I'm sorry, but I cannot help you with videos like this. You need to post the whole stack trace.

The fact that you don't do that automatically suggests that your experience level is currently under what's necessary for working on a not-small project like Tolgee.

It would be better for you to start developing your playground Spring project before trying to contribute to a project like this. It would be more effective for both sides.
